frontends/election-manager/src/components/import_cvrfiles_modal.tsx

```diff
@@ -73,14 +75,10 @@ function throwBadStatus(s: never): never {
 }
 
 export function ImportCvrFilesModal({ onClose }: Props): JSX.Element {
-  const {
-    usbDriveStatus,
-    castVoteRecordFiles,
-    electionDefinition,
-    auth,
-    logger,
-  } = useContext(AppContext);
+  const { usbDriveStatus, electionDefinition, auth, logger } =
+    useContext(AppContext);
   const addCastVoteRecordFileMutation = useAddCastVoteRecordFileMutation();
+  const cvrFiles = useCvrFilesQuery().data || [];
```

Comment:
Is it safe to use a default here? Or should we wait for data to load?

Reply:
hm, now that i think about it, probably better to block actions on waiting for data to load, since we decide which import buttons to enable based on previously imported files

Reply:
Ok, I noticed there are a number of other places that use this pattern. I didn't leave comments on all of them, but same question applies

Reply:
thanks - audited the other cases and made sure we're handling appropriately